wait/host_port.go

```diff
@@ -161,10 +162,10 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 	if err = internalCheck(ctx, internalPort, target); err != nil {
 		switch {
 		case errors.Is(err, errShellNotExecutable):
-			log.Println("Shell not executable in container, only external port validated")
+			log.Printf("Shell not executable in container, only external port validated")
```

Comment:
question: do we want to remove the `\n` here?

Reply:
Println isn't part of the interface and since log inserts a new line anyway it works.